projects/fix-machos.com/package.yml

```diff
@@ -26,7 +26,16 @@ build:
 
     - cp {{deps.zlib.net.prefix}}/lib/libz.dylib '{{prefix}}/lib'
 
-    - run: install_name_tool -change @rpath/zlib.net/v{{deps.zlib.net.version}}/lib/libz.{{deps.zlib.net.version.marketing}}.dylib {{prefix}}/lib/libz.dylib fix-machos-test
+    # zlib is weird about its lib versioning it seems
```

Comment:
what this about?

Reply:
with zlib 1.3(.0) it was linking libz-1.3.dylib. with 1.3.1 it links libz-1.3.1.dylib. it was messing up the name for changing the rpath.

Reply:
sorry, don't get you. What's it?

Reply:
The linkage was using v.marketing for zlib 1.3 and version for 1.3.1. so it broke this test, which assumed it would always be version.marketing.